deploy/cr.2.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: "pxc.percona.com/v1alpha1"
+kind: "PerconaXtraDBCluster"
+metadata:
+  name: "cluster2"
+spec:
+  secretsName: my-cluster-secrets
+  pxc:
+    size: 2
```

Comment:
oh, sorry, we can't even in example have size 2. only 3, 5 etc

Reply:
fixed